pkg/suite/suite.go

```diff
@@ -246,14 +246,17 @@ func (s *suite) Run(t *testing.T, suiteName string) {
 	})
 
 	AfterSuite(func() {
+		defer func() {
+			// We defer this to ensure it happens even if uninstalling the app fails
+			logger.Log("Deleting workload cluster")
+			err := teardown.New(state.GetFramework()).Teardown(state.GetCluster())
+			Expect(err).NotTo(HaveOccurred())
```

Comment:
Should we somehow create a slack or bigger notification that this might be hit?

Reply:
Nah, it still surfaces the failure to the test run and shows in Tekton Dashboard.